monocular_depth_estimation:

  enabled: true
  source_topic: "/wide_angle_camera_front/image_color/compressed"
  target_topic: "/wide_angle_camera_front/depth"
  model:
    model_name: "FastDepthNet"
    model_path: ""
    # model_path: "/home/nvidia/Downloads/FastDepthNet.pth"
    
    FastDepthNet:
      network: "fcn-resnet50" #fcn-mobilenet, fcn-resnet18, fcn-resnet50
      

